db/migrate/20200226201500_add_indexes.rb

```diff
@@ -0,0 +1,8 @@
+class AddIndexes < ActiveRecord::Migration[6.0]
+  def change
+    add_index :transcriptions, :group_id
+    add_index :transcriptions, :workflow_id
```

Comment:
It might be better to combine these if the the workflow id and group id are always used to filter then you can add a multi column index. Note if you ever need to filter on just one of these it has to be the first column def in the index. Also note if your query has order clauses they will need to be on the index def. 
```suggestion
    add_index :transcriptions, [:workflow_id, :group_id]
```